### ISSUE TITLE
Default global icon for invalid dial code

### ISSUE THREAD

**Issue by minhluan259** [jackocnr/intl-tel-input#1646]
### Plugin version
e.g. v23.0.10 (please try latest version)

### Steps to reproduce
1.  Set default telephone number 12345
2.  Initial intl-tel-input on page load

### Expected behaviour
Display a global icon if there is a default phone number already set with an invalid dial code.
![Screenshot_10](https://github.com/jackocnr/intl-tel-input/assets/7834473/8085680e-4476-4ba1-9c73-b6aae4d7cb85)

### Actual behaviour
Only arrow dropdown showing

**Comment by jackocnr** [comment]
If you can find the minimal set of code that reproduces the problem and put that in a CodePen, then I'd be happy to take a look. Here is [an example pen](https://codepen.io/jackocnr/pen/DyrVbw) with the plugin up and running - feel free to fork this if it helps.

**Comment by jackocnr** [comment]
It seems to work fine here: https://codepen.io/jackocnr/pen/NWVgMRp?editors=1010

So I'm not sure what the problem is. Can you reproduce it in a codepen?

**Comment by minhluan259** [comment]
The problem occurs when I use https://ipinfo.io/json

```
const input = document.querySelector("#phone");

const iti = window.intlTelInput(input, {
  utilsScript:
    "https://cdn.jsdelivr.net/npm/intl-tel-input@23.0.10/build/js/utils.js",
  initialCountry: "auto",
  nationalMode: 1,
  strictMode: 1,
  separateDialCode: 1,
  autoPlaceholder: "aggressive",
  geoIpLookup: (callback) => {
    fetch("https://ipinfo.io/json")
      .then(res => res.json())
      .then(data => callback(data.country.toLowerCase()))
      .catch(() => callback("us"));
  }
});

// store the instance variable so we can access it in the console e.g. window.iti.getNumber()
window.iti = iti;
```

**Comment by jackocnr** [comment]
Yes well it doesn't make sense to initialise the plugin with a phone number in the input, and then to set initialCountry=auto. If you already have a number, then you must already know the country, so either you (A) include the international dial code in the initial input value (recommended), or (B) use initialCountry to set the correct country from the saved number.

**Comment by minhluan259** [comment]
Initialize the plugin with a phone number in the input field in cases where we have a list of registered users with phone numbers but without country dial codes. Users will need to manually update the dial code after we integrate this plugin. Therefore, only the arrow showing may result in poor user experience.

**Comment by jackocnr** [comment]
Ok I'll take a look when I have a moment, and try to get it so it always shows either a flag or the globe icon no matter what initialisation options are used.

**Comment by chayutWQ** [comment]
react as well brother. same case

**Comment by jackocnr** [comment]
Fix released in v23.0.11 - give it a try and let me know if it works for you!

**Comment by minhluan259** [comment]
Confirmed, it works. Thank you.
